cmds/db-manager/main.go

```diff
@@ -0,0 +1,229 @@
+// Bootstrap script for Database deployment and migration
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"math"
+	"os"
+	"os/signal"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/golang-migrate/migrate/v4"
+	_ "github.com/golang-migrate/migrate/v4/database/cockroachdb"
+	_ "github.com/golang-migrate/migrate/v4/source/file"
+	dssCockroach "github.com/interuss/dss/pkg/cockroach"
+	"go.uber.org/zap"
+	"golang.org/x/mod/semver"
+)
+
+// MyMigrate is an alias for extending migrate.Migrate
+type MyMigrate struct {
+	*migrate.Migrate
+	postgresURI string
+}
+
+// Direction is an alias for int indicating the direction and steps of migration
+type Direction int
```

Comment:
nit:

add a 
```
const (
  Up Direction = 1
  Down Direction = 2
)
```

Reply:
discussed offline, I don't foresee where we are going to use this Enum